consul-l7-observability-kubernetes/step6.md

```diff
@@ -0,0 +1,26 @@
+
+Issue the following command, which will, in a separate terminal, forward the Grafana UI to port 3000 on the development host.
+
+`export POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=grafana,app.kubernetes.io/instance=grafana" -o jsonpath="{.items[0].metadata.name}") && kubectl --namespace default port-forward $POD_NAME 3000 --address 0.0.0.0`{{execute T3}}
+
+You will receive the following output.
+
+```plaintext
+Forwarding from 127.0.0.1:3000 -> 3000
+Forwarding from [::1]:3000 -> 3000
+```
+
+Now, open the Grafana UI tab next to the Terminal tab in the Katacoda environment. This will launch a new browser tab. Login with
```

Comment:
```suggestion
## Login and import a dashboard

Now, open the Grafana UI tab next to the Terminal tab in the Katacoda environment. This will launch a new browser tab. Login with
```